proc/wsmr_data_to_hskp_level0.pro

```diff
@@ -9,10 +9,11 @@
 ; use wsmr_data_to_level0.pro
 ;
 ; Inputs:
-;		FILENAME	File to process. Must be a WSMR .log file.
-;			   		Default is the 2012 Nov 2 flight data file.
 ;
 ; Keywords:	
+;		FILENAME	File to process. Must be a WSMR .log file.
+;			   		Default is the 2012 Nov 2 flight data file.
```

Comment:
Remove this comment since the 2012 flight data is currently not set as the default.

Reply:
Done!